3rdparty/openssl/include/opensslconf.h

```diff
@@ -12,193 +12,193 @@
 
 #include <openssl/opensslv.h>
 
-#ifdef __cplusplus
-extern "C"
-{
+#ifdef  __cplusplus
```

Comment:
Did these spaces automatically get inserted by the formatter?

Reply:
This file is automatically generated in update.make:
 `"perl "-I." -Mconfigdata "openssl/util/dofile.pl" "-oMakefile" "openssl/include/openssl/opensslconf.h.in" > include/opensslconf.h`
 
I think what happened is when this file was generated originally, that user ran ./scripts/format-code before committing it. I chose to use the --no-verify commit option when committing generated files in include/, include_3/, providers_3/.

I'm of the opinion that we should use OpenSSL repo generated files as is without enforcing our style guidelines on them, but I'm also happy to revert this change.